### PR TITLE
Log in form structure

### PR DESCRIPTION
**Pull request by cortezroberto**
# Description
 - We've created LogIn screen.
 - We've added Log in form to allow users log in to the app.
 - We've implemented i18n to avoid using static text.
 - We've added navigation between 'Log in' and 'Sign up' screens.
 - We've created a 'TextLink' styled component.
 - We've created a screens list in App.js to to have a better organization and a cleaner code.

  
  ## Screenshots 
Screenshot_20210805-183310

### CODE REVIEW

File: src/components/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import CheckBoxField from './CheckBox';
 import TextInputField from './TextInput';
+import TextLink from './TextLink';
 import CustomButton from './CustomButton';
 
-export {CheckBoxField, TextInputField, CustomButton};
+export {CheckBoxField, TextInputField, CustomButton,TextLink};
```
Comment (cfcortes) [inline src/components/index.js:6]: Codacy has a fix for the issue: [Insert `·`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7872340)
```suggestion
export {CheckBoxField, TextInputField, CustomButton, TextLink};
```

File: App.js [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
+import {NavigationContainer} from '@react-navigation/native';
+import {createStackNavigator} from '@react-navigation/stack';
+import {Home, LogIn, SignUp} from './src/containers';
+
+const Stack = createStackNavigator();
+
+const listScreens = [
+  {name: 'SignUp', component: SignUp},
+  {name: 'LogIn', component: LogIn},
+  // {name: 'Home', component: Home},
```
Comment (cfcortes) [inline App.js:12]: Codacy has a fix for the issue: [Delete `·`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7872340)
```suggestion
  // {name: 'Home', component: Home},
```

Comment (cfcortes) [inline App.js:12]: Codacy has a fix for the issue: [Trailing spaces not allowed.](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7872340)
```suggestion
  // {name: 'Home', component: Home},
```

File: App.js [hunk 3]
```diff
@@ -1,9 +1,29 @@
...
```
Comment (YilKanda) [inline App.js:12]: If you don't occupy this commented code, you can delete it.

File: src/components/CustomButton/styledComponent.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components/native';
 
 export const StyledButton = styled.Pressable`
-  background-color: blue;
+  background-color: grey;
```
Comment (YilKanda) [inline src/components/CustomButton/styledComponent.js:4]: You can create a color library that makes it easier for you to change them, if you require it in the future.

**Review by YilKanda — COMMENTED** [review]
It is advisable not to leave commented code.

I also recommend that you make a file for colors, where you place a range of colors that make a good combination. 

You can help yourself on sites like:

- [coolors](https://coolors.co/)
- [Adobe Color](https://color.adobe.com/es/create/color-wheel)

imagen

imagen

File: src/components/CustomButton/styledComponent.js (hunk 4 above)
Comment (cortezroberto) [reply src/components/CustomButton/styledComponent.js]: Done

File: App.js (hunk 3 above)
Comment (cortezroberto) [reply App.js]: Done

File: src/constants/colors.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+const colors = {
+    green: '#17A1A2',
+    light_green: '#38AEAF',
+    orange: '#FC8417',
+    white: '#fff',
+    dark: '#1C1C1C',
+    blue: '#0000ff',
+    red: '#ff0000',
+    gray: '#DDDDDD',
+    dark_gray: '#ABABAB',
```
Comment (cfcortes) [inline src/constants/colors.js:10]: Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7872340)
```suggestion
  dark_gray: '#ABABAB',
```

File: src/constants/colors.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
+const colors = {
+    green: '#17A1A2',
+    light_green: '#38AEAF',
+    orange: '#FC8417',
+    white: '#fff',
+    dark: '#1C1C1C',
+    blue: '#0000ff',
+    red: '#ff0000',
+    gray: '#DDDDDD',
```
Comment (cfcortes) [inline src/constants/colors.js:9]: Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7872340)
```suggestion
  gray: '#DDDDDD',
```

File: src/constants/colors.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
+const colors = {
+    green: '#17A1A2',
+    light_green: '#38AEAF',
+    orange: '#FC8417',
+    white: '#fff',
+    dark: '#1C1C1C',
+    blue: '#0000ff',
```
Comment (cfcortes) [inline src/constants/colors.js:7]: Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7872340)
```suggestion
  blue: '#0000ff',
```

File: src/constants/colors.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
+const colors = {
+    green: '#17A1A2',
+    light_green: '#38AEAF',
+    orange: '#FC8417',
+    white: '#fff',
```
Comment (cfcortes) [inline src/constants/colors.js:5]: Codacy has a fix for the issue: [Replace `····` with `··`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7872340)
```suggestion
  white: '#fff',
```

File: src/constants/colors.js [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+    dark: '#1C1C1C',
+    blue: '#0000ff',
+    red: '#ff0000',
+    gray: '#DDDDDD',
+    dark_gray: '#ABABAB',
+  };
```
Comment (cfcortes) [inline src/constants/colors.js:11]: Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7872340)
```suggestion
};
```

File: src/constants/colors.js [hunk 10]
```diff
@@ -0,0 +1,13 @@
+const colors = {
+    green: '#17A1A2',
+    light_green: '#38AEAF',
+    orange: '#FC8417',
```
Comment (cfcortes) [inline src/constants/colors.js:4]: Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7872340)
```suggestion
  orange: '#FC8417',
```

File: src/components/TextLink/styledComponent.js [hunk 11]
```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components/native';
+import colors from '../../constants/colors'
```
Comment (cfcortes) [inline src/components/TextLink/styledComponent.js:2]: Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7872340)
```suggestion
import colors from '../../constants/colors';
```

File: src/constants/colors.js [hunk 12]
```diff
@@ -0,0 +1,13 @@
...
+    white: '#fff',
+    dark: '#1C1C1C',
+    blue: '#0000ff',
+    red: '#ff0000',
+    gray: '#DDDDDD',
+    dark_gray: '#ABABAB',
+  };
+
```
Comment (cfcortes) [inline src/constants/colors.js:12]: Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7872340)
```suggestion

```

File: src/constants/colors.js [hunk 13]
```diff
@@ -0,0 +1,13 @@
+const colors = {
+    green: '#17A1A2',
+    light_green: '#38AEAF',
+    orange: '#FC8417',
+    white: '#fff',
+    dark: '#1C1C1C',
```
Comment (cfcortes) [inline src/constants/colors.js:6]: Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7872340)
```suggestion
  dark: '#1C1C1C',
```

File: src/constants/colors.js [hunk 14]
```diff
@@ -0,0 +1,13 @@
+const colors = {
+    green: '#17A1A2',
```
Comment (cfcortes) [inline src/constants/colors.js:2]: Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7872340)
```suggestion
  green: '#17A1A2',
```

File: src/constants/colors.js [hunk 15]
```diff
@@ -0,0 +1,13 @@
...
+    light_green: '#38AEAF',
+    orange: '#FC8417',
+    white: '#fff',
+    dark: '#1C1C1C',
+    blue: '#0000ff',
+    red: '#ff0000',
```
Comment (cfcortes) [inline src/constants/colors.js:8]: Codacy has a fix for the issue: [Replace `····` with `··`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7872340)
```suggestion
  red: '#ff0000',
```

File: src/constants/colors.js (hunk 12 above)
Comment (cfcortes) [inline src/constants/colors.js:12]: Codacy has a fix for the issue: [Trailing spaces not allowed.](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7872340)
```suggestion

```

File: src/constants/colors.js [hunk 16]
```diff
@@ -0,0 +1,13 @@
+const colors = {
+    green: '#17A1A2',
+    light_green: '#38AEAF',
```
Comment (cfcortes) [inline src/constants/colors.js:3]: Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7872340)
```suggestion
  light_green: '#38AEAF',
```

**Review by jkzasori — COMMENTED** [review]
I think you could add some style to your screens because are a little simple and think styles are really important. Maybe people would love your app just because has nice design...
But in general it's okay.

**Review by jkzasori — COMMENTED** [review]
Dón forget to check Codacy static code... because it say that you have some issues(18) in your code.

**Comment by cortezroberto** [comment]
> I think you could add some style to your screens because are a little simple and think styles are really important. Maybe people would love your app just because has nice design...
> But in general it's okay.

yeah, we are working in the mock up still. We decided to work on structure and functionality first then we are going to add styles.

**Comment by cortezroberto** [comment]
> Dón forget to check Codacy static code... because it say that you have some issues(18) in your code.

Well, Codacy want us to delete a space before each line of Colors.js file but for Prettier is the correct way to format it, so we think we will have to delete it so we can fix those issues.

**Review by jkzasori — APPROVED** [review]

**Review by YilKanda — APPROVED** [review]
Remember to use ESLint to correct indentation or nesting problems in your *.JS files

**Review by yovanha — APPROVED** [review]

**Pull request merged**